packages/workbox-window/src/Workbox.ts

```diff
@@ -250,7 +258,7 @@ class Workbox extends EventTargetShim {
    * @param {Object} data An object to send to the service worker
    * @return {Promise<Object>}
    */
-  async messageSW(data) {
+  async messageSW(data: {}) {
```

Comment:
Similar question for `{}` vs. `object` here: https://www.typescriptlang.org/docs/handbook/basic-types.html#object

Reply:
Updated to `object`.

I'd originally thought the Google TS style guide preferred `{}` over `object`, but upon further reading it says this:

> There are a few types related to JavaScript primitives that should never be
used: - `String`, `Boolean`, and `Number` have slightly different meaning from
the corresponding primitive types `string`, `boolean`, and `number`. Always use
the lowercase version. - `Object` has similarities to both `{}` and `object`,
but is slightly looser. Use `{}` for a type that include everything except
`null` and `undefined`, or lowercase `object` to further exclude the other
primitive types (the three mentioned above, plus `symbol` and `bigint`).